src/reducers/account.js

```diff
@@ -28,8 +28,8 @@ export default (state = initialState, action) => {
         ...state,
         signupPending: false,
         signupError: null,
-        user: action.user,
-        token: action.key,
+        user: action.response.user,
```

Comment:
I would rather the `user` and `key` come directly from action rather than getting them from the `response` object, so that if we ever want to trigger this action from something other than an API call response handler the payload can just be `{user: <user>, key: <key>}` rather than having to wrap it in a `response` object, `...{response: {user:<user>, key:<key>}}`